ansys/api/additive/v0/additive_simulation.proto

```diff
@@ -83,6 +88,10 @@ message DownloadFileResponse {
 	Progress progress = 10;	// download progress
 }
 
+message DownloadLogResponse {
```

Comment:
We don't need this. We can just return `DownloadFileResponse` from the `DownloadLogs` endpoint.